### ISSUE TITLE
aria-live="polite" not honoring aria-label

### ISSUE THREAD

**Issue by thdoan** [nvaccess/nvda#14364]
### Steps to reproduce:

1. Open nvda_test.html in the zip below

[nvda_test.zip](https://github.com/nvaccess/nvda/files/9986682/nvda_test.zip)

2. Click Toggle Modal button

### Actual behavior:

After the modal is displayed, NVDA says "times", ignoring `aria-label="Close"`.

### Expected behavior:

The expected behavior should be to honor `aria-label` and say "close", not "times".

#### NVDA installed/portable/running from source:

Portable

#### NVDA version:

2022.3.2

#### Windows version:

Windows 10 Home (build 19044.2251)

#### Name and version of other software in use when reproducing the issue:

Google Chrome Version 107.0.5304.107 (Official Build) (64-bit)

#### Other information about your system:

Using NVDA default options.

#### Does the issue still occur after restarting your computer?

Yes.

**Comment by feerrenrut** [comment]
I can reproduce this. A technical investigation is required.

**Comment by Adriani90** [comment]
For me NVDA does not report anything when activating the toggle content buttons. The focus does not move, and NVDA does not report anything. Only when I use up arro I notice the close button which means the toggle content has been activated.
